core/setup.py

```diff
@@ -69,7 +69,7 @@
         "requests<3.0.0",
         "idna>=2.5,<4",
         "cffi>=1.9,<2.0.0",
-        "pyyaml>=6.0",
+        "pyyaml>=5.3",
```

Comment:
@gshank was there a reason for this version when you added mashumaro 3?
https://github.com/dbt-labs/dbt-core/commit/5686cab5a0298be53c975805836c5cab78e981e8

Reply:
Not really. I looked at the available versions and since I had tested on a version 6.x, picked that one. I looked at the Changelogs and there hasn't been much since 5.3.